subsys/testsuite/ztest/include/ztest_test_new.h

```diff
@@ -144,6 +151,16 @@ void ztest_verify_all_test_suites_ran(void);
  */
 int z_ztest_run_test_suite(const char *name);
 
+/**
+ * @brief Returns next test within suite.
+ *
+ * @param suite Name of suite to get next test from.
+ * @param prev  Previous unit test acquired from suite, use NULL to return first
+ *		unit test.
+ * @return struct ztest_unit_test*
+ */
+struct ztest_unit_test *ztest_get_next_test(const char *suite, struct ztest_unit_test *prev);
```

Comment:
Use `z_` prefix like in the above `z_ztest_run_test_suite()` to signify that this shouldn't really be called by applications.